sql/moz-fx-data-shared-prod/telemetry_derived/feature_usage_v2/schema.yaml

```diff
@@ -398,3 +398,27 @@ fields:
 - mode: NULLABLE
   name: activitystream_sessions_both
   type: INTEGER
+- mode: NULLABLE
+  name: newtabpage_disabled
+  type: BOOLEAN
```

Comment:
Since we're here, can we add descriptions for these new fields?

Prior art: https://github.com/mozilla/bigquery-etl/blob/67119cff58df85accb0bfbf7c864a835ad5bc16d/sql/moz-fx-data-marketing-prod/iprospect/adspend_meta_v1/schema.yaml